src/client/connstring.rs

```diff
@@ -22,7 +21,7 @@ impl Host {
         }
     }
 
-    fn new_ipc(ipc: String) -> Host {
+    fn with_ipc(ipc: String) -> Host {
```

Comment:
Minor nitpick: I think what steve meant was `new_with_ipc` is more idiomatic rust.